### PR TITLE
join: add stdin example

### PR DESCRIPTION
**Pull request by yl4869**
- [*] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [*] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [*] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/join.md [hunk 1]
```diff
@@ -18,3 +18,7 @@
 - Produce a line for each unpairable line for file1:
 
 `join -a {{1}} {{file1}} {{file2}}`
+
+- Join two files, the '-' file from standard input:
+
+`join {{-}} {{file2}}`
```
Comment (navarroaxel) [inline pages/common/join.md:24]: This is an old page, now we use `path/to/file`, also use `cat` to explain the stdin stuff:
```suggestion
`cat {{path/to/file1}} | join - {{path/to/file2}}`
```

File: pages/common/join.md [hunk 2]
```diff
@@ -18,3 +18,7 @@
 - Produce a line for each unpairable line for file1:
 
 `join -a {{1}} {{file1}} {{file2}}`
+
+- Join two files, the '-' file from standard input:
```
Comment (navarroaxel) [inline pages/common/join.md:22]: Perhaps this better
```suggestion
- Join a file with data from stdin:
```

**Review by navarroaxel — COMMENTED** [review]
Hi @yl4869, welcome and thanks for your submission.

File: pages/common/join.md (hunk 1 above)
Comment (yl4869) [reply pages/common/join.md:24]: I'm happy to know it, thank you

File: pages/common/join.md (hunk 2 above)
Comment (yl4869) [reply pages/common/join.md:22]: because the '-' file can be file1 or file2, so I think know '-' from stdin is important.

Comment (yl4869) [reply pages/common/join.md:22]: may be changed  "Join '-' and file, the '-' from stdin"

Comment (CleanMachine1) [reply pages/common/join.md]: I think we could simplify this to just be
```suggestion
- Join a file from stdin:
```

**Comment by yl4869** [comment]
Ok,I get it

**Review by navarroaxel — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**